### ISSUE TITLE
Errors with php < 8

### ISSUE THREAD

**Issue by raziel057** [zircote/swagger-php#919]
I just clone the master of the project and when I try to run unit tests using PHP 7.4 I got a lot of errors due to the following commit:
https://github.com/zircote/swagger-php/commit/77fee10616d3f1ce8de34a16c3036a0d877f5cd9#diff-c3d05dc2196e7a7fb4469ace41cc580c914ec24d2272eb76920c35f2c0789b74

![image](https://user-images.githubusercontent.com/652505/112291041-1034b100-8c90-11eb-848d-2ba8cc19ff6c.png)

It comes form:
```php
if ($token[0] === T_ATTRIBUTE) {
    // consume
    $this->parseAttribute($tokens, $token, $parseContext);
    continue;
}
```

Is it expected that the master is not supported by php < 8 ?

**Comment by DerManoMann** [comment]
Looks like a bug. Its funny, though, as the build pipeline passes...
However, locally tests break for me the same way...

**Comment by michaljusiega** [comment]
See https://github.com/zircote/swagger-php/pull/857

I tried to fix same issue.

**Comment by DerManoMann** [comment]
Fixed via #924
